### PR TITLE
[16.0][MIG] base_partner_company_group

### PR DESCRIPTION
**Pull request by Highcooley**

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

**Review by cyrilmanuel — APPROVED** [review]
Not an expert but LGTM

**Review by grindtildeath — APPROVED** [review]
@OCA/crm-sales-marketing-maintainers can we please merge? :pray:

**Comment by rafaelbn** [comment]
It looks like we are missing all the commit history. Please could you review?

We should preserve the commit history like in https://github.com/OCA/partner-contact/pull/1244/commits

Isn't it?

Thank you!

**Review by rafaelbn — COMMENTED** [review]
Please @Highcooley review: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0 to preserve history, here we have a long an important one!!! really needed

**Comment by sebalix** [comment]
@rafaelbn it's because the module has been renamed, and the migration guide only retrieve the history of a given folder, not the previous one. I don't say we should not keep the previous history, but I think there is no ready-made solution in such case?

We could first retrieve the history of the original module name, then the new one (2 `git format-patch | git am` to run, and this needs to check a bit what were the previous module names).

And if a module has been split in multiple ones (I think it's the case here) it really starts to be complicated for a contributor to provide a clean history so we don't merge useless stuff from the previous module.

**Comment by simahawk** [comment]
If the module was only renamed, you could rewrite the history (I did it in the past), BUT, what I see in the migration PR is:

* a new module has been added (w/o history)
* the old one was deleted (w/ its history)

I tend to say that, since we (c2c) are the authors of the module and there are no big changes in the past... we just don't care :wink:

File: base_partner_company_group/__manifest__.py
Comment (simahawk) [inline base_partner_company_group/__manifest__.py]:
```suggestion
    "author": "Camptocamp, Odoo Community Association (OCA)",
```
@Highcooley pls fix this :pray:

**Comment by simahawk** [comment]
> If the module was only renamed, you could rewrite the history (I did it in the past), BUT, what I see in the migration PR is:
> 
>     * a new module has been added (w/o history)
> 
>     * the old one was deleted (w/ its history)
> 
> 
> I tend to say that, since we (c2c) are the authors of the module and there are no big changes in the past... we just don't care wink

Just one more point (mostly for our future selves): as I reported on #1244 `git move` should have been used. In fact, we should have blocked that PR, not this one. The authorship of some commits was not preserved. however, since there are no big changes from other authors and since the contributors added themselves to the contributors' list (I've just checked) it's kind of fine.

**Comment by simahawk** [comment]
flappy tests due to GH issues https://www.githubstatus.com/

**Pull request merged**
